documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml

```diff
@@ -0,0 +1,5 @@
+emitters:
+  "@azure-tools/cadl-autorest":
+    output-file: widgets.json
+    azure-resource-provider-folder: ../../data-plane
+    examples-directory: ./examples
```

Comment:
```suggestion
    examples-directory: ./examples
  "@azure-tools/cadl-python":
    "sdk-folder": sdk/cntosowidgetmanager/azure-contoso-widgetmanager # delete it when you run cadl compile in local
    "package-version": "1.0.0b1"
    "package-mode": "dataplane"
    "basic-setup-py": true
    "package-name": "azure-contoso-widgetmanager"
  "@azure-tools/cadl-csharp":
    "sdk-folder": sdk/cntosowidgetmanager/Azure.Contoso.WidgetManager/src # delete it and add value true @azure-tools/cadl-csharp when you run cadl compile in local
  "@azure-tools/cadl-typescript":
    "sdk-folder": sdk/cntosowidgetmanager/widgetmanager-rest # delete it when you run cadl compile in local
    generateMetadata: true # change to false if you don't want to override meta
    generateTest: true # change to false if you don't want to generate new sample test
    azureSdkForJs: true
    packageDetails:
      name: "@azure-rest/widgetmanager-rest"
      description: "Cadl Test Service"
      version: "1.0.0-beta.1"
  "@azure-tools/cadl-java":
    "sdk-folder": sdk/cntosowidgetmanager/azure-contoso-widgetmanager # delete it when you run cadl compile in local
    namespace: com.azure.cntoso.widgetmanager
```

Reply:
As we chatted I think all of this makes sense except for the versions we should not have package version info in this set of configurations.

Reply:
Updated.